mu-plugins/central-hub/tests/phpunit/unit/config-store/strStartsWith.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ *  Tests for str_starts_with()
+ *
+ * @package    spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\ConfigStore;
+
+use function KnowTheCode\ConfigStore\str_starts_with;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_StringStartsWith
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_StringStartsWith extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/internals.php';
+	}
+
+	/*
+	 *  Test should return true when substring starts with and matches haystack.
+	 */
+	public function test_should_return_true_when_substring_starts_with() {
+		$this->assertTrue( str_starts_with( 'Hello World!', 'Hello' ) );
+		$this->assertTrue( str_starts_with( 'Hello World!', 'H' ) );
+		$this->assertTrue( str_starts_with( 'Cornerstone', 'Corner' ) );
+	}
+
+	/**
+	 * Test should return false when substring does not start with & match haystack.
+	 */
+	public function test_should_return_false_when_substring_does_not_start() {
+		$this->assertFalse( str_starts_with( 'Hello World', 'World' ) );
+		$this->assertFalse( str_starts_with( 'Hello World', 'hey!' ) );
+		$this->assertFalse( str_starts_with( 'Cornerstone', 'cor' ) );
+		$this->assertFalse( str_starts_with( 'Cornerstone', 'stone' ) );
+	}
+
+	/**
+	 * Test should throw exception when string length empty.
+	 */
+	public function test_should_throw_exception_when_string_is_empty() {
+		$this->expectException( \InvalidArgumentException::class );
+		$this->expectExceptionMessage(
+			'The haystack and needle cannot be empty. Given: haystack [Cornerstone] and needle of [].'
+		);
+
+		str_starts_with( 'Cornerstone', '' );
```

Comment:
@rgadon107 Add an assertion to check that an exception is thrown when no haystack is given but a needle is and another assertion for when both are empty.  Add both of these in this test method.

Hint:  the excepted message is bound to the specific instance of invoking `str_starts_with`.  That means the one `expectExceptionMessage()`assertion on line 58 is bound to line 62.  

To add 2 more assertions, copy line 58-62 twice and then change the expected message and arguments.